BlockSettleUILib/AuthAddressDialog.cpp

```diff
@@ -202,7 +202,7 @@ void AuthAddressDialog::setAddressToVerify(const QString &addr)
    }
 }
 
-void AuthAddressDialog::setBsClient(const std::weak_ptr<BsClient> &bsClient)
+void AuthAddressDialog::init(const std::weak_ptr<BsClient>& bsClient)
```

Comment:
Why do you think that 'init' is a better name than 'setBsClient' for one-line function that set value for bsClient ?

Reply:
No reason, just touch accidentally, reverted, thank you